modules/rosa-disabling-autoscaling-nodes.adoc

```diff
@@ -6,7 +6,7 @@
 
 :_content-type: PROCEDURE
 [id="rosa-disabling-autoscaling_{context}"]
-= Disabling autoscaling nodes in an existing cluster using the rosa CLI
+= Disabling autoscaling nodes in an existing cluster using the {product-title} (ROSA) CLI (`rosa`)
```

Comment:
Just "CLI" in the title or it'll be too long; for this module, probably adjust the first paragraph so that you're giving proper product full name context there rather than in the title.

Reply:
Maybe "ROSA CLI" in title?